src/olympia/devhub/tasks.py

```diff
@@ -265,6 +268,19 @@ def handle_file_validation_result(results, file_id, *args):
     return FileValidation.from_json(file_, results)
 
 
+def insert_error_message(results, msg, msg_id):
```

Comment:
There's one other place where `compatibility_type` is set too. `description` is unused though